domain/src/test/java/org/oppia/android/domain/classify/AnswerClassificationControllerTest.kt

```diff
@@ -70,8 +69,8 @@ class AnswerClassificationControllerTest {
     private val TEST_ITEM_SELECTION_SET_1 =
       createSetOfTranslatableHtmlContentIds("content_id_0", "content_id_2", "content_id_3")
 
-    private val TEST_MULTIPLE_CHOICE_OPTION_0 = createNonNegativeInt(value = 0)
-    private val TEST_MULTIPLE_CHOICE_OPTION_1 = createNonNegativeInt(value = 1)
+    private val TEST_MULTIPLE_CHOICE_OPTION_0 = createUnsingnedInteger(value = 0)
+    private val TEST_MULTIPLE_CHOICE_OPTION_1 = createUnsingnedInteger(value = 1)
 
```

Comment:
@yash10019coder There is a typo in `createUnsignedInteger()`, also I think the original name of the method is more meaningful.

Reply:
So I shall correct the typo or change the function name to the original name @prayutsu

Reply:
You can restore the previous name.

Reply:
ok @prayutsu

Reply:
@yash10019coder The original name is `createNonNegativeInt` and not `createNonNegativeInteger`